processor/src/trace/utils.rs

```diff
@@ -315,6 +315,20 @@ impl ChipletsLengths {
         }
     }
 
+    pub fn new_debug(
+        hash_len: usize,
+        bitwise_len: usize,
+        memory_len: usize,
+        kernel_len: usize,
+    ) -> Self {
+        ChipletsLengths {
+            hash_chiplet_len: hash_len,
+            bitwise_chiplet_len: bitwise_len,
+            memory_chiplet_len: memory_len,
+            kernel_rom_len: kernel_len,
+        }
+    }
```

Comment:
I would probably rename this into `from_parts` and since we use it only for debugging purposes, I'd add `#[cfg(test)]` attribute to it.